models/task.js

```diff
@@ -0,0 +1,35 @@
+const mongoose = require("mongoose");
+
+const Schema = mongoose.Schema;
+
+const taskSchema = new Schema({
```

Comment:
Similarly here too we can store the message reference for everything. Also, rather than AssignedBy we should store AssignedTo (AssignedBy will automatically be the user who have created the message)